.github/workflows/lint.yml

```
@@ -23,21 +23,22 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v7
+        uses: actions/checkout@9c091bb21b7c1c1d1991bb908d89e4e9dddfe3e0 # v7.0.0
         with:
+          persist-credentials: false
           fetch-depth: 0
           persist-credentials: false
```

actionlint (1.7.12)

[error] key "persist-credentials" is duplicated in "with" section. previously defined at line:28,col:11. note that this key is case insensitive

(syntax-check)